src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -30,6 +32,10 @@ def __init__(
         openai_args: dict[str, str] | None = None,
         *,
         use_async: bool = False,
+        use_reasoning_summary: bool = False,
```

Comment:
Please add these to the docstring

Reply:
Thanks, added! [8409f03](https://github.com/citadel-ai/langcheck/pull/216/commits/8409f03d3efcbe23663cdd164dd36b6fe0f1c02b)